src/components/Form.js

```diff
@@ -171,3 +237,39 @@ const SubmitMessage = styled.h3`
   color: green;
   text-align: center;
 `;
+
+const CheckboxWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 0.5rem;
+
+  label {
+    border-radius: 4px;
+    border: 1px solid #394a59;
+    padding: 0.5rem;
+    text-decoration: none;
+    background: #ffffff;
+    color: #394a59;
+    cursor: pointer;
+  }
+  input:checked ~ label {
+    border-radius: 4px;
+    border: 1px solid #ffffff;
+    padding: 0.5rem;
+    text-decoration: none;
+    background: #394a59;
+    color: #ffffff;
+    cursor: pointer;
+  }
+
+  input:checked ~ label {
+    border-radius: 4px;
+    border: 1px solid #ffffff;
+    padding: 0.5rem;
+    text-decoration: none;
+    background: #394a59;
+    color: #ffffff;
+    cursor: pointer;
+  }
+`;
```

Comment:
For your styled components in general I would suggest to divide your stylings in more than three styled components, because your code is now hard to understand.